runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/WindmillTimerInternals.java

```diff
@@ -241,7 +242,7 @@ public void persistTo(Windmill.WorkItemCommitRequest.Builder outputBuilder) {
     }
 
     // Wipe the unpersisted state
-    timers.clear();
+    timerMap.clear();
```

Comment:
timerStillPresent was not cleared before, It is safe to clear both timerdata and boolen here since WindmillTimerInternals is recreated at the bundle level.